docs/00b-basics/00-basics.adoc

```diff
@@ -15,3 +15,9 @@ include::04-prerequisites-for-this-training.adoc[{include_configuration}]
 include::05-curriculum-outline.adoc[{include_configuration}]
 
 include::06-complementary-information.adoc[{include_configuration}]
+
+// tags for multilanguage support are listed here to get rid of false positives from the asciidoctor parser
+// tag::DE[]
+// end::DE[]
```

Comment:
Das ist kein False Positive, sondern eine Warnung, dass die Tags in der Datei fehlen ;P

Reply:
Es wird gewarnt, obwohl es kein Problem ist. Daher der Hinweis.